packages/theme-language-server-common/src/documents/DocumentManager.ts

```diff
@@ -181,10 +181,26 @@ export class DocumentManager {
           textDocument,
         };
       case SourceCodeType.LiquidHtml:
+        const stylesheet = sourceCode.source.match(
+          /{% stylesheet %}([\s\S]*?){% endstylesheet %}/,
```

Comment:
This feels a bit brittle? Maybe the AST visitor route would be better? Folks forget their spaces all the time.